### PR TITLE
Presentation on monitoring with the tool Prometheus

### PR DESCRIPTION
**Pull request by arxra**
No decided topic, but we promise to make it interesting!

### CODE REVIEW

**Comment by arxra** [comment]
We want to present on what monitoring is and why it matters when everything is automated.

**Comment by arxra** [comment]
Asking for topic feedback from a "random" Ta: @cesarsotovalero

**Comment by cesarsotovalero** [comment]
Hi @arxra, the scope of your presentation is too broad. 

Can you focus on one specific monitoring tool instead?

File: contributions/presentation/week2/aronber-fbjorkma/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+We want to present on why monitoring your pipelines and automations matter,
+as there seems to be an undervaluation of feedback.
+
+We aim to answer questions such as:
+
+- What is monitoring?
+- What are metrics?
+- What is the value of deploying often?
+- What is the value of the feedback from the system?
+- How to act upon the feedback gained from the system?
```
Comment (cesarsotovalero) [inline contributions/presentation/week2/aronber-fbjorkma/README.md]: The topic is too broad. Please focus on explaining all this based on a real monitoring tool.

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]

**Comment by fbjorkman** [comment]
New focus on briefly explaining what monitoring is and then focus on the monitoring tool Prometheus. Is that a better scope?

**Review by cesarsotovalero — APPROVED** [review]

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
Hi @fbjorkman.
LGTM. I'm merging your PR.
Good luck with the presentation!

**Pull request merged**